connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -340,6 +340,7 @@ public void putTaskConfigs(final @PathParam("connector") String connector,
 
     @PUT
     @Path("/{connector}/fence")
+    @Operation(hidden = true, summary = "This operation is only for inter-worker communications")
```

Comment:
Added this line to hide the newly-introduced zombie fencing API from the OpenAPI spec that we started generating with https://github.com/apache/kafka/pull/12067.